cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,32 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] PATH|RESOURCE_ALIAS(with resource_id)",
 	Short:     "Send a DELETE request",
 	Long:      "Send a DELETE request to the given path.",
 	RunE:      run,
 	ValidArgs: urls.Resources(),
 }
 
+var arguements_desc_list = map[string]string{
+	"account": `# Delete an account with ACCOUNT_ID
+  ocm delete [--parameter "deleteAssociatedResources=true"] account ACCOUNT_ID`,
```

Comment:
Where is the `deleteAssociatedResources` definied I can't find it in the ocm-cli?

Reply:
I was referring to https://api.integration.openshift.com/?urls.primaryName=Accounts%20management%20service#/default/delete_api_accounts_mgmt_v1_accounts__id_

Reply:
If we wanted to be specific we could document what query parameters are supported for each resource type.

However, looking for example at the `list clusters` CLI command we don't list the specific supported parameters so I am not sure whether it is worth it.

I would like other views around it. Any preference about this @etabak ?